buildSrc/src/main/kotlin/boudicca-springboot-app.gradle.kts

```diff
@@ -18,10 +18,28 @@ dependencies {
     testImplementation(versionCatalog.findLibrary("spring-boot-starter-test").get())
 }
 
+val containerEngine: String by rootProject.extra
+
 task<Exec>("imageBuild") {
     inputs.file("src/main/docker/Dockerfile")
     inputs.files(tasks.named("bootJar")) //TODO extract to own plugin
     dependsOn(tasks.named("assemble"))
-    commandLine("docker", "build", "-t", "localhost/boudicca-${project.name}", "-f", "src/main/docker/Dockerfile", ".")
+    commandLine(containerEngine, "build", "-t", "localhost/boudicca-${project.name}", "-f", "src/main/docker/Dockerfile", ".")
 }
 
+task<Exec>("imageBuildMultiplatform") {
```

Comment:
please also add this image build task to the eventcollectors, they sadly still have their own task because they are not a spring boot app (till now at least, something to consider in the future...)